src/main/java/com/internet/shop/controllers/shoppingcart/DeleteProductFromTheCartController.java

```diff
@@ -23,8 +23,9 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         Long productId = Long.valueOf(req.getParameter("id"));
+        Long userId = (Long) req.getSession().getAttribute(USER_ID);
```

Comment:
You can use the already defined constant(USER_ID) of the "AddToCartController" class so as not to create a new constant in every controller

Reply:
agree, genius